packages/office-ui-fabric-react/.vscode/settings.json

```diff
@@ -23,6 +23,7 @@
     "**/node_modules": true,
     "**/lib": true,
     "**/lib-amd": true,
+    "**/lib-es2015": true,
```

Comment:
comment: normally not really a factor unless you open this package directly..

Reply:
That's typically what I do, since I don't work in Fabric much and mainly just want the core files available for reference (as part of a VS Code workspace) while working on internal code